src/config.ts

```diff
@@ -131,6 +131,7 @@ export async function resolveConnections(config: Configuration, resolver?: strin
       client,
       connection: {
         host: (connection as any).host,
+        server: (connection as any).server,
```

Comment:
Maybe we need to type things better. Let's try to avoid `connection as any` in here.